### ISSUE TITLE
Level: Lock Smith

### ISSUE THREAD

**Issue by tttppp** [andybalaam/rabbit-escape#510]
```
:name=Lock Smith
:description=Can you slot the rabbits inbetween rabbots?
:author_name=tttppp
:author_url=https://github.com/tttppp
:solution.1.code=m'0'V''3,HV'm#"''VV*'HHfm*|{']ff3v3*f"0"!,#m']|"H'#0&
:num_rabbits=3
:num_to_save=3
:rabbit_delay=4
:bridge=5
:dig=1
#Q       #
#        #
#        #
#        #
#        #
MM#MMMMMMM
MM  MMMMMM
MMM MMMMMM
MMtttttOtM
M**MMMMMtM
MM)yyyyy*M
MMMMMMMMMM
:*=ty
:*=(y
:*=ty
```

I think this is a little bit different.

https://github.com/andybalaam/rabbit-escape/blob/master/rabbit-escape-engine/src/rabbitescape/levels/development/rabbots/Lock-Smith.rel

**Comment by tttppp** [comment]
BTW The [level editor](http://artificialworlds.net/rabbit-escape/level-editor/) is great for getting started!

**Comment by colonelfazackerley** [comment]
18 attempts. This is an evil level. Not quite the most malicious.

xEOxEO-dgO[l ____3_Q!dl&gcQ[_cJ[E&__d&

I think I enjoyed it. I was certainly relieved when I did it. 

My solution has a different character. I wonder if @tttppp will object to my last token placement...

**Comment by tttppp** [comment]
It's like "UFO" all over again!

I think multiple solutions to this is fine. If we ever implement the token drop shield then we could shield off that approach.

**Comment by colonelfazackerley** [comment]
It's still hard, I dont think the level needs a drop shield.

**Comment by andybalaam** [comment]
Fiendish!
```
v@vnvv"v)m_vEm9p7cnbbxE_F)@Fp
```

**Comment by andybalaam** [comment]
Wow, both your solutions are different from mine, and very cool.  I love the fact that this is like a real lock, in that there are different ways to pick it that are all valid because they fit the mechanism.

**Comment by andybalaam** [comment]
#497 is now fixed, so moved into staging.

**Comment by andybalaam** [comment]
Released in 0.11
